src/Standards/Generic/Sniffs/PHP/DeclareStrictTypesSniff.php

```diff
@@ -310,7 +310,7 @@ public function process(File $phpcsFile, $stackPtr)
 
                 $this->checkOtherDeclarations($phpcsFile, $next);
 
-                return (count($tokens) + 1);
+                return $phpcsFile->numTokens;
```

Comment:
If I remember correctly, elsewhere `return ($phpcsFile->numTokens + 1)` is used. Shouldn't really make a difference, but just thought I'd mention it for consistency sake.